loadtest/control/simplecontroller/actions.go

```diff
@@ -47,6 +47,11 @@ func (c *SimpleController) sendDirectMessage(userID string) control.UserStatus {
 }
 
 func (c *SimpleController) scrollChannel(u user.User) control.UserActionResponse {
+	collapsedThreads, resp := control.CollapsedThreadsEnabled(u)
+	if resp != nil {
```

Comment:
If we opt to return value like other actions this should be something like this:
```suggestion
	if resp.Err != nil {
```